src/resourcemanager/resourcemanager.cpp

```diff
@@ -0,0 +1,479 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <algorithm>
+#include <filesystem>
+#include <fstream>
+
+#include <utils/exception.hpp>
+#include <utils/json.hpp>
+
+#include "logger/logmodule.hpp"
+#include "resourcemanager.hpp"
+
+namespace aos::sm::resourcemanager {
+
+using common::utils::CaseInsensitiveObjectWrapper;
+using common::utils::GetArrayValue;
+
+/***********************************************************************************************************************
```

Comment:
We should switch to using unnamed namespace instead of statics. Need to discuss with team..